src/OV/OpenVarioMenu.cpp

```diff
@@ -99,6 +99,101 @@ FileMenuWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
   });
 }
 
+class ScreenSSHWidget final
+  : public RowFormWidget
+{
+  UI::Display &display;
+  UI::EventQueue &event_queue;
+
+public:
+  ScreenSSHWidget(UI::Display &_display, UI::EventQueue &_event_queue,
+                 const DialogLook &look) noexcept
+    :RowFormWidget(look),
+     display(_display), event_queue(_event_queue) {}
+
+private:
+  /* virtual methods from class Widget */
+  void Prepare(ContainerWindow &parent,
+               const PixelRect &rc) noexcept override;
+};
+
+void
+ScreenSSHWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
+                         [[maybe_unused]] const PixelRect &rc) noexcept
+{
+  AddButton("Enable", [](){
+    static constexpr const char *argv[] = {
+      "/bin/sh", "-c", 
+      "systemctl enable dropbear.socket && printf '\nSSH has been enabled'", 
```

Comment:
No shell scripts acceptable. Will never be merged.

Reply:
how can I enable or disable a daemon like SSH or variod without systemctl using C++ code? systemctl is a shell command